### PR TITLE
feat: add divideNum

### PR DESCRIPTION
**Pull request by willcaul**
https://github.com/char0n/ramda-adjunct/issues/482

### CODE REVIEW

File: src/divideBy.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { divide, flip } from 'ramda';
+
+/**
+ * Divides two numbers, where the second number is divided by the first number.
+ *
+ * @func divideBy
```
Comment (char0n) [inline src/divideBy.js]: I'm struggling with the name of the function. In original ticket I've suggested to use `divideNum`. The thing is that `By` and `With` suffixes on functions are reserved for special behavior described in [this issue](https://github.com/char0n/ramda-adjunct/issues/653). By looking at the name of the function I'd expect that it consumes unary function as a first argument. I'd prefer `divideNum` even though it's "uglier" then `divideBy`.

File: test/divideBy.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+describe('divideBy', function() {
+  it('should divide the second number by the first number', function() {
+    eq(RA.divideBy(2, 1), 0.5);
+    eq(RA.divideBy(2.0, 1.0), 0.5);
+    eq(RA.divideBy(0, 2.0), Infinity);
+    eq(RA.divideBy(0.0, 2.0), Infinity);
+    eq(RA.divideBy(-0.0, 2.0), -Infinity);
+  });
+
+  it('should curry', function() {
```
Comment (char0n) [inline test/divideBy.js]: To satisfy test for currying we need to test for function call combinations. For binary function, it's two cases:
```js
it('should curry', function() {
  eq(RA.divideBy(2)(1), 0.5);
  eq(RA.divideBy(2, 1), 0.5);
});

File: src/divideBy.js (hunk 1 above)
Comment (willcaul) [reply src/divideBy.js]: That makes sense... Thanks for the explanation.

**Comment by willcaul** [comment]
@char0n - thanks for the PR comments... updated.

Regarding the name, I changed it from `divideBy` to `divideNum` as requested. But I was wondering... you think `divideByNum` would be clearer, or do you think it is clearer as-is?

**Comment by char0n** [comment]
@willcaul not sure at this point, I think what matters is that `divideNum` and `subtractNum` are named by same convention and you can expect the same behavior. In future if we come up with better names, we can still alias the functions with new names and leave the old names for backward compatibility.

**Pull request merged**